stack-9.2.1.yaml

```diff
@@ -102,7 +102,7 @@ flags:
     importLens: false
     moduleName: true
     ormolu: false
-    qualifyImportedNames: false
+    qualifyImportedNames: true
```

Comment:
we can remove the flag also for stack, as the default value is true

Reply:
Oh, I see. Will do.